lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
I'm traveling at the moment but will look into this when I reach my destination this evening. Simply removing this conditional should present the error when viewing a site using wagon locally. I believe that's where I saw the issue crop up but I can go back and look through my local commits to see exactly when I encountered this error. I suppose that's one problem with squashing commits. Thanks, @did.